src/mpid/common/hcoll/subconfigure.m4

```diff
@@ -1,19 +1,14 @@
 [#] start of __file__
 
 AC_DEFUN([PAC_SUBCFG_PREREQ_]PAC_SUBCFG_AUTO_SUFFIX,[
-	AC_ARG_ENABLE(hcoll,
-	    AC_HELP_STRING([--enable-hcoll], [Enable HCOLL collectives support.]))
-
 	PAC_SET_HEADER_LIB_PATH(hcoll)
 	PAC_PUSH_FLAG(LIBS)
-	if test "$enable_hcoll" != "no"; then
+	if test ! -z "$with_hcoll" -a "$with_hcoll" != "no"; then
```

Comment:
Why does `$with_hcoll` need to be not-NULL?  What is it's already present in the system path?  I should be able to simply pass `--with-hcoll` to enable it, isn't it?

Reply:
Yes, that sounds like a better test. I'll revise.

Reply:
Oh, I think what I was trying to accomplish was to make sure configure would fail if the user passed `--with-hcoll=[yes|path/to/install]`. If the user didn't supply `--with-hcoll` at all, then configure should just disable it an move on. I think my logic was still a bit wrong and will revise in any case.